docs/get-started/system-requirements.md

```diff
@@ -0,0 +1,19 @@
+---
+title: System requirements
+description: System requirements to sync and run Teku
+sidebar_position: 1
+---
+
+# System requirements
+
+The following are the minimum system requirements for running a full node, with Teku as a beacon node and validator client:
+
+- CPU: 4 cores at 2.8 GHz
+- Memory: 16 GB RAM
+- Storage: SSD with 2 TB free space
+- Operating system: 64-bit Linux, Mac OS X, or Windows
+- Internet: Broadband internet connection (10 Mbps)
+- Power: Uninterruptible power supply (UPS)
```

Comment:
UPS is a bit extreme. The way the inactivities penalties work means that even if you go offline for a couple of hours, you can quickly recoup that being online later. The only tragedy happens if you are offline when it is time to propose a block or during a sync committee participation, but both are very rare.

So maybe we add a note that this is recommended but not a requirement?